client/src/components/navbar/navbar.jsx

```diff
@@ -147,15 +142,15 @@ function MyAccount({ items, label, Logout }) {
       {items.map((item, index) => {
         if (item.label === "Logout") {
           return (
-            <MenuItem onClick={() => Logout(history)} key={item}>
+            <MenuItem onClick={() => Logout(history)} key={index}>
```

Comment:
I have read that it's not best practice to use only the index as the key. I found the following: 

> It's a bad idea to use the array index since it doesn't uniquely identify your elements. In cases where the array is sorted or an element is added to the beginning of the array, the index will be changed even though the element representing that index may be the same. This results in unnecessary renders.

Reply:
Resolved. Used label value instead of index value.